generators/client-2/templates/src/main/webapp/app/account/reset/request/_reset-request.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, OnInit, Inject, Renderer, ElementRef} from '@angular/core';
+import { TranslatePipe } from '../../../shared/translate.pipe';
+
+export class PasswordResetInitComponent implements OnInit {
```

Comment:
you need the `@Component` annotation with template and selector here. You dont have to declare pipes in it but the annotation is still needed